File: JNItest/src/Calculadora.cpp
#include <Calculadora.h>
Calculadora::Calculadora(){
	valA=0;
	valB=0;
}
Calculadora::Calculadora(int a, int b){
	valA=a;
	valB=b;
}
void Calculadora::setValA(int a){
	valA=a;
}
void Calculadora::setValB(int b){
	valB=b;
}
int Calculadora::suma(){
	return valA+valB;
}
int Calculadora::resta(){
	return valA-valB;
}
int Calculadora::multiplica(){
	return valA*valB;
}

File: springConfiguration/src/main/java/com/mx/eglobal/Calculadora.java
package com.mx.eglobal;

/**
 * 
 *
 */
public class Calculadora{
    private int valA;
    private int valB;
    private Evaluator evalData;


    public Calculadora(){/*{{{*/
        this.valA = 0;
        this.valB = 0;
    }/*}}}*/
    public Calculadora(int valA, int  valB){/*{{{*/
        this.valA = valA;
        this.valB = valB;
    }/*}}}*/

    public void setEvalData( Evaluator evalData ){/*{{{*/
        this.evalData = evalData;
    }/*}}}*/
    public Evaluator getEvalData(){/*{{{*/
        return this.evalData;
    }/*}}}*/
    public int suma(){/*{{{*/
        return valA + valB;
    }/*}}}*/
    public int resta(){/*{{{*/
        return valA - valB;
    }/*}}}*/
    public int multiplicacion(){/*{{{*/
        return valA * valB;
    }/*}}}*/

    public void setValA(int valA ){/*{{{*/
        this.valA = valA;    
    }/*}}}*/
    public void setValB(int valB ){/*{{{*/
        this.valB = valB;    
    }/*}}}*/


    public String toString(){
        return "ValA: " + this.valA + " -- ValB: " + this.valB;
    }

}
